lib/full.css

```diff
@@ -14,6 +14,29 @@
   color: <<FORE_PRIMARY>>;
 }
 
+.sj-search-box-0,
```

Comment:
Can we somehow use selectors that don't use `-X`

Those selectors tend to be the ones that break the most frequently